packaging/windows_installer.nsi.in

```diff
@@ -108,7 +108,7 @@ Section Uninstall
   SetShellVarContext all
 
   ${FreeOrion_DLL_LIST_UNINSTALL}
-  Delete "$$INSTDIR\Python27.zip"
+  Delete "$$INSTDIR\Python${FreeOrion_PYTHON_VERSION}.zip"
```

Comment:
You need to apply the same change to line 63 too.

Reply:
Fixed.